tests/unit/test_cursorManager.py

```diff
@@ -47,6 +49,32 @@ def test_unselectPrevChar(self):
 		cm.script_selectCharacter_back(None) # "a" unselected
 		self.assertEqual(cm.selectionOffsets, (0, 0)) # Caret at "a", no selection
 
+	def test_unselectNextChar(self):

```

Comment:
In fact, the later tests use the term "forward" and "back" rather than "next" and "prev". It would be nice if we could be consistent here, I think forward and back make more sense since they are used in the name of the `script_` functions.